recipes/scikit-learn-intelex/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "scikit-learn-intelex" %}
+{% set version = "2021.2.1" %}
+{% set buildnumber = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/intel/scikit-learn-intelex/archive/{{ version }}.tar.gz
+  sha256: 5cfd513245e686d5503ae6ef2d5a17dbf5d0ca41d14b2322a492e1f816ece251
+
+build:
+  skip: true  # [osx]
+  number: {{ buildnumber }}
+  script: "{{ PYTHON }} setup_sklearnex.py install --single-version-externally-managed --record=record.txt"
+
+requirements:
+  host:
+    - python
+    - setuptools
+  run:
+    - python
+    - daal4py ==2020.3.1
+    - scikit-learn >=0.21
+
+test:
+  requires:
+    - pandas
+    - pytest
```

Why are these two required? I don't see them used down below.
```suggestion
    - pandas
    - pytest
```